src/datagrid/DataGridValues.tsx

```diff
@@ -210,35 +230,146 @@ export class GridActions extends React.PureComponent<any, GridActionsState> {
+            {columnName: "Name", cellData: "Georgia"},
+            {columnName: "Creation Date", cellData: "Sep 11, 2008"},
+            {columnName: "Favorite color", cellData: "Blue"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 16166},
+            {columnName: "Name", cellData: "Brynn"},
+            {columnName: "Creation Date", cellData: "Aug 2, 2014"},
+            {columnName: "Favorite color", cellData: "Orange"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 30574},
+            {columnName: "Name", cellData: "Brad"},
+            {columnName: "Creation Date", cellData: "Jan 4, 2019"},
+            {columnName: "Favorite color", cellData: "Yellow"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 2459},
+            {columnName: "Name", cellData: "Beverly"},
+            {columnName: "Creation Date", cellData: "Mar 2, 2019"},
+            {columnName: "Favorite color", cellData: "Pink"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 14262},
+            {columnName: "Name", cellData: "Johnson"},
+            {columnName: "Creation Date", cellData: "Jun 23, 2019"},
+            {columnName: "Favorite color", cellData: "Blue"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 14262},
+            {columnName: "Name", cellData: "Johnson"},
+            {columnName: "Creation Date", cellData: "Jun 23, 2019"},
+            {columnName: "Favorite color", cellData: "Blue"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 59729},
+            {columnName: "Name", cellData: "Sibyl"},
+            {columnName: "Creation Date", cellData: "Feb 27, 2016"},
+            {columnName: "Favorite color", cellData: "Red"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 55316},
+            {columnName: "Name", cellData: "Debby"},
+            {columnName: "Creation Date", cellData: "Mar 27, 2019"},
+            {columnName: "Favorite color", cellData: "Blue"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 92422},
+            {columnName: "Name", cellData: "Roslyn"},
+            {columnName: "Creation Date", cellData: "Apr 26, 2016"},
+            {columnName: "Favorite color", cellData: "Blue"},
+        ],
+    },
+    {
+        rowData: [
+            {columnName: "User ID", cellData: 83943},
+            {columnName: "Name", cellData: "Lottie"},
+            {columnName: "Creation Date", cellData: "Jun 21, 2018"},
+            {columnName: "Favorite color", cellData: "Yellow"},
+        ],
+    },
+];
+
+// Function to get data for page based on pagenumber
+export const getPageData = (pageIndex: number, pageSize: number): Promise<DataGridRow[]> => {
```

Comment:
Query: Why is the getPageData outside datagrid ? Is there any customisation needed in getPageData ?

Reply:
This function is used in stories. It is not part of datagrid code.